server/models/questions/forkQuestion.js

```diff
@@ -43,8 +43,8 @@ function forkQuestion({ username, questionId }) {
             const forkedQuestionId = results.insertId
 
             connection.query(
-              `INSERT INTO questions_editors(question_id, editor) VALUES(?,?)`,
-              [forkedQuestionId, username],
+              `INSERT INTO questions_editors(question_id, editor, access) VALUES(?,?,?)`,
+              [forkedQuestionId, username, 'write'],
```

Comment:
#### CHANGE

1. The person forking will be `creator` so he will also have `write` access